### DISCUSSION TITLE
Verslunarstjórinn - 2023/7/2

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#34] [vefsida-ummæli]
# Verslunarstjórinn - 2023/7/2

Jóhann er orðinn verslunarstjórinn

https://biskupsreglan-git-chromebook-matthiasgudni.vercel.app/faerslur/frettir/verslunarstjorinn

**Comment by Ronaldo-is-better** [comment]
Þetta er mjög flott, auðvitað er Rónaldo besti fótboltamaður mannkynsögunnar.

**Comment by TJ478** [comment]
Vó, TJ-inn er að commenta á vercel appinu.
